navi/navi/navi/SheetView.swift

```diff
@@ -15,31 +15,32 @@ struct SheetView: View {
     @Environment(\.dismiss) var dismiss
     var body: some View {
         
-        
-        VStack {
-            
-            Toggle("Ramp", isOn: $rampCheck)
-                .toggleStyle(SwitchToggleStyle(tint: .red))
-            Toggle("Elevator", isOn: $elevatorCheck)
-                .toggleStyle(SwitchToggleStyle(tint: .red))
-            Toggle("Accessible Gate", isOn: $agCheck)
-                .toggleStyle(SwitchToggleStyle(tint: .red))
-            if rampCheck {
-                Text("Ramp")
-            }
-            if elevatorCheck {
-                Text("Elevator")
-            }
-            if agCheck {
-                Text("Accessible Gate")
+            VStack {
+                
+                Toggle("Ramp", isOn: $rampCheck)
+                    .toggleStyle(SwitchToggleStyle(tint: .red))
+                Toggle("Elevator", isOn: $elevatorCheck)
+                    .toggleStyle(SwitchToggleStyle(tint: .red))
+                Toggle("Accessible Gate", isOn: $agCheck)
+                    .toggleStyle(SwitchToggleStyle(tint: .red))
```

Comment:
As a next step, you could try persisting these values to UserDefaults. An easy way of doing this would be to use the [@AppStorage property wrapper](https://www.hackingwithswift.com/quick-start/swiftui/what-is-the-appstorage-property-wrapper) on the definitions of the variables.